File: Programming Basics with Python/01 First Steps in Coding/More exercises/06_Fishland.py
mackerel_price = float(input())
sprat_price = float(input())
bonito_kg = float(input())
horse_mackerel_kg = float(input())
mussels_kg = float(input())
bonito_price = mackerel_price * 1.6
horse_mackerel_price = sprat_price * 1.8
mussels_price = 7.5
money = bonito_kg * bonito_price + horse_mackerel_kg * horse_mackerel_price + mussels_kg * mussels_price
print(f'{money:.2f}')

File: JS Basics/01 More Exercises Simple Operations and Calculations/06. Fishland.js
function fishland(input1, input2, input3, input4, input5) {
    let mackerelPrice = Number(input1)
    let spratPrice = Number(input2)
    let bonitoKg = Number(input3)
    let scadKg = Number(input4)
    let musselsKg = Number(input5)
    let bonitoPrice = mackerelPrice * 1.6
    let scadPrice = spratPrice * 1.8
    let moneyNeeded = bonitoKg * bonitoPrice + musselsKg * 7.5 + scadKg * scadPrice
    console.log(moneyNeeded.toFixed(2))
}
fishland(5.55, 3.57, 4.3, 3.6, 7)